posts/2020-11-30-open-liberty-devfile-stack.adoc

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: "Develop Open Liberty apps directly in OpenShift"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/awisniew90
+author_github: https://github.com/awiniew90
+seo-title: Develop Open Liberty apps directly in OpenShift - OpenLiberty.io
+seo-description: Get true-to-production development directly in a Kubernetes / OpenShift cluster
+blog_description: "Get true-to-production development directly in a Kubernetes / OpenShift cluster"
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Develop Open Liberty apps directly in OpenShift
+Adam Wisniewski <https://github.com/awisniew90>
+
+True-to-production development is critical when creating cloud applications, and what better way to develop those apps than directly in the cloud! The link:https://github.com/OpenLiberty/application-stack[Java Open Liberty devfile stack] allows you to do just that by providing a simple yet robust Open Liberty development experience directly in a Kubernetes / OpenShift cluster. It does the heavy lifting allowing you to focus on your code as if it were running right on your laptop. 
```

Comment:
I think somewhere.. either upfront or as a footnote at the end, we should address generic k8s.   At the least we say it's easy but an extra step to create the ingress config.   Possibly we might even link to the odo.dev article.